constants/src/gen/cpp/generate_constants.cc

```diff
@@ -66,16 +66,17 @@ int main(int /* argc */, char ** /* argv */) {
 
   CONST(SSL_OP_CIPHER_SERVER_PREFERENCE);
   CONST(SSL_OP_NO_TICKET);
-  CONST(SSL_OP_NO_SSLv3);
-  CONST(SSL_OP_NO_TLSv1);
-  CONST(SSL_OP_NO_TLSv1_1);
-  CONST(SSL_OP_NO_TLSv1_2);
+  CONST(SSL_OP_NO_SSLv3);  // Only for testing
```

Comment:
Looks like the remaining tests don't actually care what flag you're using, right? Want to just switch them to SSL_OP_NO_TICKET or something?

(Only reason I mention is we'll probably be deleting SSL 3.0 support altogether sometime this month. At that point I imagine we'll set SSL_OP_NO_SSLv3 to zero.)

Reply:
Sure, sounds good.